File: BaekJoon/src/step5/Problem_1065.java
package step5;
import java.util.Scanner;

public class Problem_1065 {
	int ari_seq(int n) {
		int cnt = 0;
		int n1 = 0;
		int n2 = 0;
		int n3 = 0;
		
		for(int i=1;i<=n;i++) {
			if(i<100) {
				cnt += 1;
				continue;
			} else {
				n1 = i/100;
				n2 = (i-n1*100)/10;
				n3 = i%10;
				
				if(n2-n1 == n3-n2) {
					cnt++;
				}
			}
		}
				
		return cnt;
	}
	
	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		int N = in.nextInt();
		
		Problem_1065 pro = new Problem_1065();
		int cnt = pro.ari_seq(N);
		
		System.out.println(cnt);
		
		in.close();
	}

}

File: BaekJoon/src/step5/Problem_1065.py
N = int(input())

def ari_seq(n):
    cnt = 0
    n1 = 0
    n2 = 0
    n3 = 0

    for i in range(1,n+1):
        if i <100:
            cnt += 1
        else:
            n1 = i//100
            n2 = (i - n1*100)//10
            n3 = i%10

                        
            if n2-n1 == n3-n2:
                cnt += 1

    return cnt


cnt = ari_seq(N)
print(cnt)
